e2e/sim_test.go

```
@@ -435,7 +430,7 @@ func TestAppSimulationAfterImport(t *testing.T) {
 	)
 	require.Equal(t, "SimApp", newApp.Name())
 
-	newApp.InitChain(abci.RequestInitChain{
+	newApp.InitChain(&abci.RequestInitChain{
```

golangci-lint (1.62.2)

Error return value of `newApp.InitChain` is not checked

(errcheck)